partcad/src/partcad/user_config.py

```
@@ -10,143 +10,110 @@
         config_path = os.path.join(
             UserConfig.get_config_dir(),
             "config.yaml",
         )
         if os.path.exists(config_path):
             try:
-                self.config_obj = yaml.safe_load(open(config_path, "r"))
-                if self.config_obj is None:
-                    self.config_obj = {}
+                with open(config_path, "r") as f:
+                    self.read_config(f)
             except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)